bin/gwdetchar-scattering

```diff
@@ -497,6 +501,15 @@ for i, channel in enumerate(sorted(alldata[0].keys())):
 # close panel group
 page.div.close()
 
+# write a csv file.  
+logger.debug('Writing a summary CSV record')
+
+actives = actives.coalesce() # merge contigous segments 
+gps = [x for x in highsnrtrigs[names[0]] for y in actives if x in y]
+segs =  [y for x in highsnrtrigs[names[0]] for y in actives if x in y]
+data = Table([gps, segs], names = ['trigger_time','segment'])
```

Comment:
@siddharth101, just a couple of minor formatting changes:

```suggestion
data = Table([gps, segs], names=['trigger_time', 'segment'])
```

Reply:
Yup. Should have done that already.